packages/mui-material/src/FormControlLabel/FormControlLabel.js

```diff
@@ -167,10 +167,12 @@ const FormControlLabel = React.forwardRef(function FormControlLabel(inProps, ref
       {React.cloneElement(control, controlProps)}
       {required ? (
         <Stack direction="row" alignItems="center">
```

Comment:
```suggestion
        <Stack display="block">
```
I agree with @amitabh-sahu's [suggestion](https://github.com/mui/material-ui/pull/39721#pullrequestreview-1715423055)